Cubical/Functions/Embedding.agda

```diff
@@ -186,3 +199,137 @@ Embedding-into-isSet→isSet (f , hasPropFibers-f) isSet-B x y p q =
     cong-f-p≡cong-f-q = isSet-B (f x) (f y) (cong f p) (cong f q)
     isEquiv-cong-f = hasPropFibers→isEmbedding hasPropFibers-f x y
     cong-f⁻¹ = invIsEq isEquiv-cong-f
+
+embedIsProp : {f : A → B}
+            → isEmbedding f
+            → isProp B
+            → isProp A
+embedIsProp isEmb isPropB a b = equivFun (invEquiv (_ , isEmb a b)) (isPropB _ _)
+
+universeEmbedding :
```

Comment:
What is the idea behind this name?

Reply:
It comes from the fact that `F` is an embedding between two universes, `Type ℓ` and `Type ℓ₁`